apps/webapp/Dockerfile.slim

```
@@ -6,56 +1,57 @@
+FROM alpine AS runner
+ARG PROJECT
 
-# Don't run production as root
 RUN addgroup --system --gid 1001 nodejs
-RUN adduser --system --uid 1001 nextjs
-
-# Copy necessary files
-COPY --from=installer /app/apps/webapp/.next/standalone ./
-COPY --from=installer /app/apps/webapp/.next/static ./apps/webapp/.next/static
-COPY --from=installer /app/apps/webapp/public ./apps/webapp/public
+RUN adduser --system --uid 1001 nodejs
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)